iOS/Flows/Conversation/Expressions/ExpressionViewController.swift

```diff
@@ -1,31 +1,193 @@
 //
-//  ExpressionViewController.swift
+//  ExpressionCaptureViewController.swift
 //  Jibber
 //
-//  Created by Benji Dodgson on 3/25/22.
+//  Created by Benji Dodgson on 4/29/22.
 //  Copyright © 2022 Benjamin Dodgson. All rights reserved.
 //
 
 import Foundation
 import Combine
-import Localization
 
-class ExpressionViewController: EmojiPickerViewController {
+class ExpressionViewController: ViewController {
```

Comment:
This class is both layered with the expression capture vc on top, and emotions on the bottom, and flip back and forth depending on the interaction. works really well surprisingly.